scraper.rb

```diff
@@ -18,13 +18,17 @@ def scrape(h)
   klass.new(response: Scraped::Request.new(url: url).response)
 end
 
+CEASED_MEMBER_IDS = %w(1756605 1756928 1757002).freeze
+ceased_member_urls = CEASED_MEMBER_IDS.map { |id| "http://www.duma.gov.ru/structure/deputies/#{id}/" }
+
 url = 'http://www.duma.gov.ru/structure/deputies/?letter=%D0%92%D1%81%D0%B5'
-page = scrape(url => AllMembersPage)
-warn "Found #{page.members.count} members"
+current_member_urls = scrape(url => AllMembersPage).member_urls
+member_urls = current_member_urls + ceased_member_urls
+warn "Found #{member_urls.count} members"
 
 ScraperWiki.sqliteexecute('DELETE FROM data') rescue nil
-page.members.each do |mem|
-  data = mem.merge scrape(mem[:source] => MemberPage).to_h
+member_urls.each do |mem_url|
+  member_page = scrape(mem_url => MemberPage)
   # puts data
```

Comment:
This comment has now drifted out of date, largely as there _is_ no `data` any more. In general, though, I think it's better to construct the data first and then push it to ScraperWiki, rather than having the `save` call also perform logic (even if it's just calling `.to_h`). Plus, that also gives us the option of the debugging output, which is useful to check that nothing is changing unexpectedly etc.